allauth/templates/account/email/password_changed_notification_message.txt

```diff
@@ -0,0 +1,6 @@
+{% extends "account/email/base_notification.txt" %}
+{% load i18n %}
+
+{% block content %}{% blocktrans %}Your Password has been changed
+
+Changes made by {{ username }}.{% endblocktrans %}{% endblock content %}
```

Comment:
I don't think it makes sense here to include "Changes made by...", as it will always be the user him/herself, so it does not add anything.